mitodl/open-discussions
# Remove bootcamps, csail, mitpe, see platform courses

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #3747 

#### What's this PR do?
- Removes code for importing courses from the following platforms: bootcamps, csail, see, mitpe.  Videos from these platforms (specifically bootcamps and csail) are still okay.
- Adds a management command for deleting all courses from specified platforms


#### How should this be manually tested?
On master branch, you can import some courses from the affected platforms:
- Set the following in your .env file:
  ```
  CSAIL_BASE_URL=https://cap.csail.mit.edu/
  ````
- Run these commands:  
  ```
  python manage.py backpopulate_csail_data
  python manage.py backpopulate_bootcamps_data
  ```
- Go to http://localhost:8063/learn/search?type=course, both "Bootcamps" and "CSAIL" should be present under the "Offered By" filter with >0 courses each.
- On this branch, run the following to get rid of them:
  ```
  python manage.py delete_courses csail bootcamps
  ```
- It should complete successfully and indicate that x courses were deleted for each platform. 
- Go to http://localhost:8063/learn/search?type=course again, neither "Bootcamps" nor "CSAIL" should be present under the "Offered By" filter.
